sdk/healthinsights/azure-healthinsights-radiologyinsights/generated_tests/test_radiology_insights.py

```diff
@@ -0,0 +1,474 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+import pytest
+from devtools_testutils import recorded_by_proxy
+from testpreparer import RadiologyInsightsClientTestBase, RadiologyInsightsPreparer
+
+
+@pytest.mark.skip("you may need to update the auto-generated test case before run it")
```

Comment:
It doesn't look like these tests under generated_tests/ actually run? Given you have other tests under tests/ is there a reason to keep these?